development-docs/TESTING.md

```diff
@@ -77,8 +73,7 @@ to add the second option. Rest of these approaches are not so essential.
 ## Test Phases
 
 In general, we use classic test phases: `setup`, `exercise`, `verify` and `teardown`. Every phase will be described for the
-`src.test.java.io.strimzi.kafka.bridge.http` package. It is very similar with `src.test.java.io.strimzi.kafka.bridge.amqp` only
-one difference is that `http` package using Strimzi `Kafka` container by `test-containers` and `amqp` package using KafkaFacade.
+`src.test.java.io.strimzi.kafka.bridge.http` package.
```

Comment:
I guess the actual package is `io.strimzi.kafka.bridge.http`?

Reply:
Yes you are right, I think the intent of who wrote this testing doc was about highlighting the full path under the "test" but as you said the package name is what you wrote. I rephrased a little bit.